packages/util/__tests__/index.js

```diff
@@ -485,43 +489,45 @@ test('normalizeHttpResponse should not change response', async (t) => {
     response: { headers: {} }
   }
   normalizeHttpResponse(request)
-  t.deepEqual(request, { response: { headers: {} } })
+  t.deepEqual(request, { response: { statusCode: 500, headers: {} } })
 })
 test('normalizeHttpResponse should update headers in response', async (t) => {
   const request = {
     response: {}
   }
   normalizeHttpResponse(request)
-  t.deepEqual(request, { response: { headers: {}, body: {} } })
+  t.deepEqual(request, { response: { statusCode: 200, headers: {}, body: {} } })
 })
 
 test('normalizeHttpResponse should update undefined response', async (t) => {
   const request = {}
   normalizeHttpResponse(request)
-  t.deepEqual(request, { response: { headers: {} } })
+  t.deepEqual(request, { response: { statusCode: 500, headers: {} } })
 })
 
 test('normalizeHttpResponse should update nullish response', async (t) => {
```

Comment:
not sure if I'm missing it, but let's add a test that asserts the response if only part of the response params are presnet (e.g. only body or only status code)

Reply:
added